packages/fractor-typoscript/tests/TypoScriptFileProcessor/config/fractor.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+use a9f\Fractor\Configuration\FractorConfiguration;
+use a9f\FractorTypoScript\Configuration\TypoScriptProcessorOption;
+use a9f\FractorTypoScript\Tests\Fixtures\DummyTypoScriptFractorRule;
+use Helmich\TypoScriptParser\Parser\Printer\PrettyPrinterConfiguration;
+
+return FractorConfiguration::configure()
+    ->withOptions([
+        TypoScriptProcessorOption::INDENT_SIZE => 4,
+        TypoScriptProcessorOption::INDENT_CHARACTER => PrettyPrinterConfiguration::INDENTATION_STYLE_SPACES,
+        //TypoScriptProcessorOption::INDENT_CHARACTER => 'auto',
```

Comment:
This should be removed.

Reply:
I kept this for documentation purpose.

Reply:
Yeah, it does not exactly look like documentation :smile:  What was the purpose? Reminding someone that "auto" is valid as well? Then we should phrase that as a sentence that people can actually understand :blush: